Versions-mac.plist.in

```diff
@@ -17,8 +17,6 @@
 			<string>10.13</string>
 			<string>10.14</string>
 			<string>10.15</string>
-			<!-- <string>10.16</string> --> <!-- Reverted by Xcode 12 GM -->
-			<!-- <string>11.0</string> -->
```

Comment:
Not sure we if we should add the comment or not. This was what git auto merge did.